apps/test/unit/applab/ExporterTest.js

```diff
@@ -75,7 +75,7 @@ describe('The Exporter,', function () {
     });
   });
 
-  describe("when exporting,", function () {
+  xdescribe("when exporting,", function () {
```

Comment:
Yes. This feature is far from done, so I'll reenable the tests when I
finish the feature.

On Wednesday, June 1, 2016, Brent Van Minnen <notifications@github.com>
wrote:

> In apps/test/unit/applab/ExporterTest.js
> <https://github.com/code-dot-org/code-dot-org/pull/8479#discussion_r65436479>
> :
>
> > @@ -75,7 +75,7 @@ describe('The Exporter,', function () {
> >      });
> >    });
> >
> > -  describe("when exporting,", function () {
> > +  xdescribe("when exporting,", function () {
>
> xit? xdescribe? is that just disabling these tests?
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/8479/files/80648d7e6a71a750d5f498817e77e640c51d2a59#r65436479>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAHnmw8NTuzyccQo1lz3IAw8RbL7e8dxks5qHewZgaJpZM4Ihhot>
> .
>